apps/api/src/services/configuration.service.ts

```diff
@@ -26,7 +26,8 @@ export class ConfigurationService {
       RAKUTEN_RAPID_API_KEY: str({ default: '' }),
       REDIS_HOST: str({ default: 'localhost' }),
       REDIS_PORT: port({ default: 6379 }),
-      ROOT_URL: str({ default: 'http://localhost:4200' })
+      ROOT_URL: str({ default: 'http://localhost:4200' }),
+      WEB_AUTH_RP_ID: host({ default: 'localhost' })
```

Comment:
What should the value be in the cloud?

Reply:
The domain name: ghostfolio.io
We may also be able to omit it as specified here: https://www.w3.org/TR/webauthn-2/#relying-party-identifier